apps/document-storage/src/server/app/blob-store/service.js

```diff
@@ -93,5 +90,32 @@ export async function uploadBlob(
 export async function downloadBlob(blobName) {
 	const blobContent = await getBlockBlobClient(blobName).downloadToBuffer();
 
-	return blobContent
+	return blobContent;
+}
+
+/**
+ * @param {{documents: object}[]} documents
+ */
+export async function documentsCreateUrl(documents) {
+	documents.map((document) => {
+		if (document.caseType.match('application')) {
+			document.blobStoreUrl = buildApplicationURL(
+				document.caseReference,
+				document.GUID,
+				document.documentName
+			);
+		}
+		return documents;
```

Comment:
added twice to pass linting checks - is there another way around this?

Reply:
What does the error say?

Reply:
it says the .map needs a return statement. but then I need an additional return statement for the function

Reply:
I think it would make more sense here to return `document` instead of `documents`. https://www.w3schools.com/jsref/jsref_map.asp